test/services/teams/helper/helper.user.js

```diff
@@ -5,13 +5,17 @@ const accountModel = require('../../../../src/services/account/model.js');
 // const app = require(SRC + 'app');
 const { ObjectId } = require('mongoose').Types;
 const app = require('../../../../src/app');
-const { warning } = require('../../../../src/logger');
 
-const PASSWORD = (process.env.TEST_PW || '').trim();
-if (PASSWORD === '') warning('process.env.TEST_PW is not defined');
-const PASSWORD_HASH = (process.env.TEST_HASH || '').trim();
-if (PASSWORD_HASH === '') warning('process.env.TEST_HASH is not defined');
-const AT = '@schul-cloud.org';
+const {
+	TEST_PW,
+	TEST_HASH,
+	AT,
```

Comment:
AT als global const finde ich zu viel des guten. Das wird nur hier in dem Test für das zusammenbauen der email genutzt. Sobald wir dort auch testObject umstellen fliegt das AT komplett raus.